server/chat-plugins/room-events.ts

```diff
@@ -58,11 +87,16 @@ export const commands: ChatCommands = {
 			}
 			if (!this.runBroadcast()) return;
 			const hasAliases = getAllAliases(room).length > 0;
+			const hasCategories = getAllCategories(room).length > 0;
 
 			let buff = '<table border="1" cellspacing="0" cellpadding="3">';
-			buff += `<th>Event Name:</th>${hasAliases ? `<th>Event Aliases:</th>` : ``}<th>Event Description:</th><th>Event Date:</th>`;
-			for (const i in room.settings.events) {
-				buff += formatEvent(room.settings.events[i], hasAliases);
+			buff += '<th>Event Name:</th>';
+			if (hasAliases) buff += '<th>Event Aliases:</th>';
+			if (hasCategories) buff += '<th>Event Categories:</th>';
+			buff += '<th>Event Description:</th><th>Event Date:</th>';
+
+			for (const event of Object.values(room.settings.events).filter(e => 'eventName' in e).map(e => e as RoomEvent)) {
+				buff += formatEvent(room, event, hasAliases, hasCategories);
 			}
```

Comment:
```suggestion
			for (const event of Object.values(room.settings.events)) {
				if ('eventName' in event) {
					buff += formatEvent(room, event, hasAliases, hasCategories);
				}
			}
```

I suspect you might want a `getAllEvents` to go with your other functions...